### ISSUE TITLE
How many points to use?

### ISSUE THREAD

**Issue by linhaojia13** [lyqun/FPConv#10]
Hi, thank you for your impressive work!
I  notice that the paper said "Each block is of size 2m by 2m, containing 11k'', however, the default num_points configuration in the training script 'train_scannet.sh' is 8192. 
It seems that you have explored the configuration of num_points and block_size on ScanNet v2, haven't you?
If so, could you please share your experience about this preprocessing configuration on ScanNet as well as on S3DIS?

Have a nice day!

**Comment by lyqun** [comment]
Yes, configuration of 2x2 with 11k points is our best experiment setting before our paper submission. After that, we found a tiny version, which is shown in this repo for scannet training. It is for quickly reproducing the results, but suffers about 1 mIoU point loss. If you wanna reproduce our results, I will provide the full model file. Please leave your email address here.

Note that, for the full model, it may cost a very long time for training, which is not good for conducting experiments comparison.

**Comment by linhaojia13** [comment]
My email address is 771235732@qq.com.

Thank you very much!

**Comment by Sungmin-Woo** [comment]
Hi, @lyqun and @linhaojia13 !
Could you provide the full model file?
I hope I could reproduce the original results.
